ap_api/resources/quotes/index.mdx

```diff
@@ -5,7 +5,11 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
-The quote endpoint is used to provide potential trading pairs and exchange rates for a given asset pair. For example, if a user wants to trade their USDC for XLM, the quote endpoint would provide them with the current exchange rate for that trade.
+The `/quotes` endpoints are used to fetch exchange rates previously provided to users via the [SEP-38 RFQ API][sep-38] implemented by the Anchor Platform. Quotes are commitments to trade an off-chain asset such as fiat euro for an on-chain asset such as USDC, or vice-versa, at a specified rate. Quotes are only valid for a specified period of time.
+
+For each request for a quote from a client application such as a wallet or exchange, the Anchor Platform makes a `GET /rate` request to the business server. The business then returns the exchange rate, quote expiration, and any additional fees for facilitating the transaction back to the Anchor Platform.
```

Comment:
Wording suggestion for clarity, take it or leave it.

"For each quote request from a client application such as a wallet or exchange, the Anchor Platform makes a GET /rate request to the business server. The business then returns the exchange rate, quote expiration, and any additional fees for facilitating the transaction back to the Anchor Platform."